appserver/extras/arquillian-containers/payara-micro/src/main/java/fish/payara/arquillian/container/payara/managed/PayaraMicroDeployableContainer.java

```diff
@@ -209,6 +209,11 @@ public ProtocolMetaData deploy(Archive<?> archive) throws DeploymentException {
                 cmd.add(1, "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5006");
             }
 
+            // If autoBindHttp has been disabled, remove the option
+            if (!configuration.isAutoBindHttp()) {
+                cmd.removeIf(option -> option.equals("--autoBindHttp"));
```

Comment:
I think it's easier/more consistent only add the options if `isAutoBindHttp` is true.  Especially since `"--autoBindRange", "1000"` won't be removed now.

I do love the usage of `removeIf` ;)